README.md

```diff
@@ -1,4 +1,32 @@
 Compose Switch 
 --------------
 
-This is a thin wrapper to actually run either docker-compose v1 (python) or docker compose v2 (compose CLI plugin) depending on user's configuration
+Compose Switch is a replacement to the Compose V1 `docker-compose` (python) executable. It translate the command line info Compose V2 `docker compose` then run the latter.
+
+## installation
+
+We provide an install script for automated installtion:
+
+```console
+$ curl -fL https://raw.githubusercontent.com/docker/compose-cli/main/scripts/install/install_linux.sh | sh
+```
+
+### Manual installation
+
+1. download compose-switch binary for your architecture
+```console
+$ curl -fL https://github.com/docker/compose-switch/releases/download/v1.0.1/docker-compose-linux-amd64 -o /usr/local/bin/compose-switch
+```
+1. make compose-switch executable
+```console
+$ chmode +x /usr/local/bin/compose-switch
+```
+1. rename docker-compose if you already have it installed as `/usr/local/bin/docker-compose` 
+```console
+$ mv /usr/local/bin/docker-compose /usr/local/bin/docker-compose-v1
```

Comment:
Is it possible not to rename it?

Reply:
we need to define alternative `docker-compose` command that will come first in `PATH`, and unfortunately `/usr/local/bin` comes first (we should not use /usr/local/sbin`)

Reply:
Ha, this is not the one from the package, but one that could have been installed by the user

Reply:
yes, as documented by https://docs.docker.com/compose/install/